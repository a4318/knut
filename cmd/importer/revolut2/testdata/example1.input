Type,Product,Started Date,Completed Date,Description,Amount,Fee,Currency,State,Balance
CARD_PAYMENT,Current,2020-07-01 16:35:02,2020-07-02 05:27:33,a,-16.95,1.00,CHF,COMPLETED,779.65
CARD_PAYMENT,Current,2020-07-02 10:39:51,2020-07-03 04:32:46,b,-31.80,0.00,CHF,COMPLETED,747.85
CARD_PAYMENT,Current,2020-07-02 12:03:28,2020-07-03 04:32:46,b,-6.00,0.00,CHF,COMPLETED,741.85
CARD_PAYMENT,Current,2020-07-02 23:35:49,2020-07-03 07:23:45,c,-3.00,0.00,CHF,COMPLETED,738.85
CARD_PAYMENT,Current,2020-07-06 11:11:20,2020-07-07 04:23:31,d,-17.95,0.00,CHF,COMPLETED,720.90
CARD_PAYMENT,Current,2020-07-06 22:38:13,2020-07-07 23:49:36,e,-39.51,0.00,CHF,COMPLETED,681.39
CARD_PAYMENT,Current,2020-07-07 13:13:54,2020-07-08 04:45:32,d,-35.90,0.00,CHF,COMPLETED,645.49
CARD_PAYMENT,Current,2020-07-12 11:52:10,2020-07-13 09:02:08,f,-35.90,0.00,CHF,COMPLETED,609.59
CARD_PAYMENT,Current,2020-07-16 16:21:53,2020-07-19 02:56:14,g,-11.85,0.00,CHF,COMPLETED,597.74
CARD_PAYMENT,Current,2020-07-22 16:41:59,2020-07-23 01:01:46,h,-43.90,0.00,CHF,COMPLETED,553.84
CARD_PAYMENT,Current,2020-07-22 18:10:36,2020-07-23 01:01:46,b,-5.00,0.00,CHF,COMPLETED,548.84
CARD_PAYMENT,Current,2020-07-26 12:54:33,2020-07-27 04:30:17,i,-19.90,0.00,CHF,COMPLETED,528.94
CARD_PAYMENT,Current,2020-07-25 08:34:25,2020-07-27 13:43:26,j,-4.60,0.00,CHF,COMPLETED,524.34
CARD_PAYMENT,Current,2020-07-30 12:41:36,2020-07-31 05:51:30,k,-35.90,0.00,CHF,COMPLETED,488.44
TOPUP,Current,2020-08-04 00:03:38,2020-08-04 00:03:38,l,2000.00,0.00,CHF,COMPLETED,2488.44
CARD_PAYMENT,Current,2020-08-03 18:37:23,2020-08-04 01:51:10,b,-5.00,0.00,CHF,COMPLETED,2483.44
CARD_PAYMENT,Current,2020-08-03 12:48:21,2020-08-04 01:51:10,m,-1.00,0.00,CHF,COMPLETED,2482.44
CARD_PAYMENT,Current,2020-08-03 17:17:56,2020-08-04 07:26:32,n,-95.96,0.00,CHF,COMPLETED,2386.48
EXCHANGE,Current,2020-08-05 10:59:52,2020-08-05 10:59:52,o,-1293.25,0.00,CHF,COMPLETED,1093.23
