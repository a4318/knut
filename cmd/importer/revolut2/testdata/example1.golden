2020-07-02 "a"
Assets:Accounts:Revolut Expenses:TBD                 16.95 CHF
Assets:Accounts:Revolut Expenses:Fees                    1 CHF

2020-07-02 balance Assets:Accounts:Revolut 779.65 CHF

2020-07-03 "b"
Assets:Accounts:Revolut Expenses:TBD                  31.8 CHF

2020-07-03 "b"
Assets:Accounts:Revolut Expenses:TBD                     6 CHF

2020-07-03 "c"
Assets:Accounts:Revolut Expenses:TBD                     3 CHF

2020-07-03 balance Assets:Accounts:Revolut 738.85 CHF

2020-07-07 "d"
Assets:Accounts:Revolut Expenses:TBD                 17.95 CHF

2020-07-07 "e"
Assets:Accounts:Revolut Expenses:TBD                 39.51 CHF

2020-07-07 balance Assets:Accounts:Revolut 681.39 CHF

2020-07-08 "d"
Assets:Accounts:Revolut Expenses:TBD                  35.9 CHF

2020-07-08 balance Assets:Accounts:Revolut 645.49 CHF

2020-07-13 "f"
Assets:Accounts:Revolut Expenses:TBD                  35.9 CHF

2020-07-13 balance Assets:Accounts:Revolut 609.59 CHF

2020-07-19 "g"
Assets:Accounts:Revolut Expenses:TBD                 11.85 CHF

2020-07-19 balance Assets:Accounts:Revolut 597.74 CHF

2020-07-23 "b"
Assets:Accounts:Revolut Expenses:TBD                     5 CHF

2020-07-23 "h"
Assets:Accounts:Revolut Expenses:TBD                  43.9 CHF

2020-07-23 balance Assets:Accounts:Revolut 548.84 CHF

2020-07-27 "i"
Assets:Accounts:Revolut Expenses:TBD                  19.9 CHF

2020-07-27 "j"
Assets:Accounts:Revolut Expenses:TBD                   4.6 CHF

2020-07-27 balance Assets:Accounts:Revolut 524.34 CHF

2020-07-31 "k"
Assets:Accounts:Revolut Expenses:TBD                  35.9 CHF

2020-07-31 balance Assets:Accounts:Revolut 488.44 CHF

2020-08-04 "b"
Assets:Accounts:Revolut Expenses:TBD                     5 CHF

2020-08-04 "l"
Expenses:TBD            Assets:Accounts:Revolut       2000 CHF

2020-08-04 "m"
Assets:Accounts:Revolut Expenses:TBD                     1 CHF

2020-08-04 "n"
Assets:Accounts:Revolut Expenses:TBD                 95.96 CHF

2020-08-04 balance Assets:Accounts:Revolut 2386.48 CHF

2020-08-05 "o"
Assets:Accounts:Revolut Expenses:TBD               1293.25 CHF

2020-08-05 balance Assets:Accounts:Revolut 1093.23 CHF

